tests/templates/kuttl/opa-authorization/00-rbac.yaml.j2

```diff
@@ -0,0 +1,29 @@
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: use-integration-tests-scc
+rules:
+{% if test_scenario['values']['openshift'] == "true" %}
+  - apiGroups: ["security.openshift.io"]
+    resources: ["securitycontextconstraints"]
+    resourceNames: ["privileged"]
+    verbs: ["use"]
+{% endif %}
```

Comment:
Good to have this here, can you also please run the tests on OpenShift to make sure the changes are OpenShift compatible?

Reply:
fixed in 2f6057c88ed1efcdc080df433169fe7d34d77278 and successfully tested on OpenShift 4.12